docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -88,6 +88,8 @@ JavaScript Syntax: @param {type} name _description_
 To denote a custom function parameter as optional:
 * Put square brackets around the parameter name. For example: `@param {string} [text] Optional text`.
 
+Note that if you are using optional parameters and the data is not provided by Excel, the parameter by default will be populated with a `null` value.
```

Comment:
Consider simplifying this sentence to something like "The default value of optional parameters is null."